memor/prompt.py

```diff
@@ -265,9 +264,9 @@ def load(self, file_path: str) -> None:
         with open(file_path, "r") as file:
             self.from_json(file.read())
 
-    def from_json(self, json_object: Union[str, Dict[str, Any]]) -> None:
+    def _validate_extract_json(self, json_object: Union[str, Dict[str, Any]]) -> None:
```

Comment:
@sadrasabouri Why did you remove `@staticmethod`? When I mentioned internal method, I meant internal static method. Please remove `self` and revert `@staticmethod` for all of them.

Reply:
fixed in 2a8f385.